packages/fast-components-react-base/src/button/button.props.ts

```diff
@@ -8,11 +9,15 @@ export enum ButtonHTMLTags {
     button = "button"
 }
 
-export interface IButtonProps {
+export interface IButtonHandledProps {
     /**
      * The HTML tag (defaults to ButtonHTMLTags.button)
      */
     tag?: ButtonHTMLTags;
 
```

Comment:
since you're in here...do you want to add a comment?

Reply:
I'll take care of it in another PR - I want to get this in before other components go in.